RadixWallet/Core/DesignSystem/HitTargetSize.swift

```diff
@@ -9,6 +9,9 @@ public enum HitTargetSize: CGFloat {
 	/// 34
 	case smaller = 34
 
+	/// 40
+	case extraSmall = 40
```

Comment:
:D

Reply:
Getting harder to make up names, this needs an overhaul soon